codecov.yml

```diff
@@ -6,6 +6,7 @@
 # the coverage.
 #
 codecov:
+  token: fb52015a-b78e-4d89-a6e0-a4829aea2efa
```

Comment:
we should not require to add a token for open source project.

I think that the codecov token errors are due to other root causes.

codecov was working without a token for a long time.

Reply:
According to the documentation, it's not required.

In practice, there's a race condition on codecov's side, some buggy logic which seems to disproportionately affect projects with larger numbers of github actions statuses that report coverage.

Before we added this coverage was failing something like 40% of builds on this PR; after adding it, I haven't seen one.